mlir/include/mlir/Dialect/Mesh/IR/MeshOps.td

```diff
@@ -41,7 +41,7 @@ def Mesh_ClusterOp : Mesh_Op<"cluster", [Symbol]> {
     determine the layout and the addressing space of the computation distributed
     across the mesh.
 
-    3. `dim_sizes`: This attribute represents the shape of the device cluster.
+    3. `shape`: This attribute represents the shape of the device cluster.
```

Comment:
We should also remove the doc for `rank` and renumber the rest?

Reply:
Yes, thank you. I removed it.